### PR TITLE
Wizard recipe: Check-v0.15.1

### PR DESCRIPTION
**Pull request by bramtayl**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: Check
* Version: v0.15.1

@staticfloat please review and merge.

### CODE REVIEW

File: C/Check/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd check-0.15.1
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make
+make install
+exit
```
Comment (giordano) [inline C/Check/build_tarballs.jl:20]:
```suggestion
install_license COPYING.LESSER
```

Comment (bramtayl) [reply C/Check/build_tarballs.jl:20]: What is this for?

Comment (giordano) [reply C/Check/build_tarballs.jl:20]: https://juliapackaging.github.io/BinaryBuilder.jl/dev/build_tips/#Installing-the-license-file

Comment (bramtayl) [reply C/Check/build_tarballs.jl:20]: Thanks!

**Pull request closed without merging**